core/ir/opnd_shared.c

```diff
@@ -1386,6 +1386,59 @@ opnd_replace_reg(opnd_t *opnd, reg_id_t old_reg, reg_id_t new_reg)
     }
 }
 
+opnd_t
+opnd_create_increment_reg(opnd_t opnd, uint increment)
+{
+    opnd_t inc_opnd DR_IF_DEBUG(= { 0 }); /* FIXME: Needed until i#417 is fixed. */
+    CLIENT_ASSERT(opnd_is_reg(opnd), "opnd_create_increment_reg: not a register");
+
+    reg_id_t reg = opnd.value.reg_and_element_size.reg;
+    reg_id_t min_reg = DR_REG_INVALID;
+    reg_id_t max_reg = DR_REG_INVALID;
+#ifdef AARCH64
```

Comment:
Looks like a significant chunk of this function, its main initialising code, is only compiled in for AArch64! Won't it fail for any architecture  other than AArch64? Currently none of the non-AArch64 regression tests call it, hence the full precommit pass.

As it's a fairly specific AArch64 SVE function it's probably best to `#ifdef` the whole function. Perhaps consider calling it `opnd_create_increment_sve_reg()` or something?